pegasus/sites.v3/code.org/public/yourschool.haml

```diff
@@ -5,11 +5,25 @@ social:
   "og:title": "Does your school teach computer science?"
   "og:description": "Expand computer science at your school or district. Join the thousands of schools who have already incorporated high quality computer science education into their curriculum and provide opportunities for the students in your local area."
 ---
+- require 'country_codes'
+- require 'state_abbr'
 %script{src: "https://cdn.optimizely.com/js/400912536.js"}
 - census_announcement = DCDO.get('census_announcement', nil)
 - if 'us' != hoc_detect_country
   - census_announcement ||= {}
   - census_announcement['hide_map'] = 'true'
+- if dashboard_user && dashboard_user[:school_info_id]
+  - census_announcement ||= {}
+  - census_announcement['user_name'] = dashboard_user[:name] if dashboard_user[:name]
+  - census_announcement['user_email'] = dashboard_user[:email] if dashboard_user[:email]
+  - census_announcement['is_teacher'] = 'true' if dashboard_user[:user_type] == 'teacher'
+  - school_info = DASHBOARD_DB[:school_infos].where(id: dashboard_user[:school_info_id]).first
```

Comment:
Not a huge fan of `DASHBOARD_DB` calls inside Haml views.  I know we talked this through already, but prefer an AJAX call to a Dashboard API, so we're not reaching into the Dashboard DB from random Pegasus pages.